protocol/cmd/dydxprotocold/cmd/root.go

```diff
@@ -63,11 +68,18 @@ func NewRootCmd(
 	option *RootCmdOption,
 	homeDir string,
 ) *cobra.Command {
+	logger := log.NewLogger(os.Stdout)
 	return NewRootCmdWithInterceptors(
 		option,
 		homeDir,
 		func(serverCtxPtr *server.Context) {
```

Comment:
This "interceptor" is run [here](https://github.com/dydxprotocol/v4-chain/blob/dc316d839ba583921637bfde35b52a482e2c29a3/protocol/cmd/dydxprotocold/cmd/root.go#L158), which happens after the `config.toml` is loaded.